MANUAL.txt

```diff
@@ -3251,7 +3260,7 @@ If you just want a regular inline image, just make sure it is not
 the only thing in the paragraph. One way to do this is to insert a
 nonbreaking space after the image:
 
-    ![This image won't be a figure](/url/of/image.png)\ 
```

Comment:
It's pretty important that this space be here at the end of line. I assume your editor removed it automatically.